content/dart/concepts/map/map.md

```diff
@@ -0,0 +1,38 @@
+---
+Title: 'Map'
+Description: 'The Map object is a simple key/value pair. Keys and values in a map may be of any type. A Map is a dynamic collection. In other words, Maps can grow and shrink at runtime.'
+Subjects:
+  - 'Map term in Dart'
```

Comment:
Only use the subjects present in this [file](https://github.com/Codecademy/docs/blob/main/documentation/subjects.md)